src/MLJInterface.jl

```diff
@@ -321,7 +323,9 @@ end
 function unwrap_units_single(v::AbstractVector, ::Type{D}) where {D}
     return v, D()
 end
-function unwrap_units_single(v::AbstractVector{T}, ::Type{D}) where {D,T<:UnionAbstractQuantity}
+function unwrap_units_single(
+    v::AbstractVector{T}, ::Type{D}
+) where {D,T<:UnionAbstractQuantity}
```

Comment:
I think this type might not be correct here. We also want to capture mixes of quantities and numbers... 

(But perhaps the new promotion rules would account for this?)